lib/utils.js

```diff
@@ -291,6 +291,9 @@ exports.autoStoreData = function (req, res, next) {
 
 exports.handleCookies = function (app) {
   return function handleCookies (req, res, next) {
+    // do not show cookie banner on cookie consent page
+    app.locals.shouldShowNewCookieMessage = req.path !== '/cookie-consent'
```

Comment:
I think this is a scoping issue as `app.locals` is essentially a global object.  There's a theoretical race condition here.

As this is a demo of the banner I wonder whether we should just set this variable in the views themselves, or set up routes which render with a model.